docs/deployment/image_vector.md

```diff
@@ -13,16 +13,47 @@ images:
   repository: gcr.io/google_containers/pause-amd64
   tag: "3.0"
   version: 1.17.x
+  architecture: "amd64"
 - name: pause-container
   sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
   repository: gcr.io/google_containers/pause-amd64
   tag: "3.1"
   version: ">= 1.18"
+  architecture: "amd64"
 ...
 ```
 
 That means that the Gardenlet will use the `pause-container` in with tag `3.0` for all seed/shoot clusters with Kubernetes version `1.17.x`, and tag `3.1` for all clusters with Kubernetes `>= 1.18`.
 
+## Image Vector Architecture
+
+```yaml
+images:
+- name: pause-container
+  sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
+  repository: k8s.gcr.io/pause
+  tag: "3.0"
+  architecture: "amd64"
+- name: pause-container
+  sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
+  repository: k8s.gcr.io/pause
+  tag: "3.0"
+  architecture: "arm64"
+- name: pause-container
+  sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
+  repository: k8s.gcr.io/pause
+  tag: "3.0"
+  architecture: "*"
+...
+```
+
+Architecture is an optional field of image. It specifes CPU architecture of machines on which this image can be used. There are three valid option for architecture field as follow:
+ - `amd64` : This sepecifies image can run only on machines having CPU architecture `amd64`.
+ - `arm64` : This sepecifies image can run only on machines having CPU architecture `arm64`.
+ - `*` : This sepecifies image can run on machines having CPU architecture either `amd64` or `arm64`.
+
+If image doesn't specifies any architecture then by default it's architecture is considered `*`.
```

Comment:
- `amd64`: This specifies image can run only on machines having CPU architecture `amd64`.
 - `arm64`: This specifies image can run only on machines having CPU architecture `arm64`.
 - `*`: This specifies image can run on machines having CPU architecture either `amd64` or `arm64`.

If image doesn't specifies any architecture then by default it's architecture is considered `*`.